src/Centreon/Domain/Exception/EntityNotFoundException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Centreon\Domain\Exception;
+
+use Throwable;
+
+class EntityNotFoundException extends \Exception
+{
+    public function __construct($message = "", Throwable $previous = null)
```

Comment:
prototype is not the same than the parent ?

Reply:
Yes, this error code is automatically set to 404.